ivy/src/main/contraband-scala/sbt/librarymanagement/LibraryManagementCodec.scala

```diff
@@ -0,0 +1,17 @@
+/**
+ * This code is generated using [[http://www.scala-sbt.org/contraband/ sbt-contraband]].
+ */
+
+// DO NOT EDIT MANUALLY
+package sbt.librarymanagement
+trait LibraryManagementCodec extends sbt.internal.librarymanagement.formats.GlobalLockFormat
```

Comment:
This is a duplicate `sbt.librarymanagement.LibraryManagementCodec`. Should be moved or elided.